index.html

```diff
@@ -2,5 +2,6 @@
     <head></head>
     <body>
         Hei Oslomet og data1200
+        <p>lorem ipsum yada yada latin</p>
```

Comment:
this isent what we discussed you were supposed to translate the message... speak to me tomorrow....

Reply:
okay boss... im sorry....